nixos/modules/services/x11/window-managers/dwm.nix

```diff
@@ -25,12 +45,12 @@ in
       { name = "dwm";
         start =
           ''
-            dwm &
+            ${cfg.package}/bin/dwm &
```

Comment:
If an user has installed dwm, then the users dwm should be started. So I would prefer to start dwm without absolute path.

Reply:
Isn't that against what other NixOS services and window managers do?

I'm fine making the change for backwards compatibility though. It just seems non-standard.

Reply:
I have removed the explicit reference to dwm. It is now _very_ similar to https://github.com/NixOS/nixpkgs/pull/94185.

I'll let you choose which one to merge. I'll close mine if the other is merged.